Yank/utils.py

```diff
@@ -1090,16 +1091,24 @@ def net_charge(self, value):
 
 # OpenEye functions
 # ------------------
-def is_openeye_installed():
+def is_openeye_installed(oetools=('oechem', 'oequacpac', 'oeiupac', 'oeomega')):
+    # Complete list of module: License check
+    tools_license = {'oechem':'OEChemIsLicensed',
+                     'oequacpac': 'OEQuacPacIsLicensed',
+                     'oeiupac': 'OEIUPACIsLicensed',
+                     'oeomega': 'OEOmegaIsLicensed'}
+    tool_keys = tools_license.keys()
+    # Cast oetools to tuple if its a single string
+    if type(oetools) is str:
+        oetools = (oetools,)
     try:
-        from openeye import oechem
-        from openeye import oequacpac
-        from openeye import oeiupac
-        from openeye import oeomega
-
-        if not (oechem.OEChemIsLicensed() and oequacpac.OEQuacPacIsLicensed()
-                and oeiupac.OEIUPACIsLicensed() and oeomega.OEOmegaIsLicensed()):
-            raise ImportError
+        for tool in oetools:
+            if tool in tool_keys:
```

Comment:
If the input `oetools` is a single string, I would agree, but because you can feed in multiple tools, I have to iterate through each individual tool, then check them against the set of valid tools. I then need to act on each tool individually. If I only needed the check, I could just look for a non-strict subset of the two lists